receiver/sqlqueryreceiver/testdata/integration/sqlserver/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
Can we use a more recent version of the SQL Server docker image?

Reply:
I changed version to 2022-latest